newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -116,6 +116,13 @@ export default class SearchPanel extends PureComponent<Props, State> {
               }
               hintText={t`Text to search in parameters`}
               onChange={(e, searchText) => this.setState({ searchText })}
+              onKeyPress={event => {
+                if (event.charCode === 13) {
+                  //enter key pressed
+                  if (this.searchTextField) this.searchTextField.blur();
```

Comment:
I would avoid to blur something, instead see if you can focus something else :)

Reply:
That was just to remove focus on the text-field. Is it fine if I just remove the line? It is not that important